mars/worker/__main__.py

```diff
@@ -83,6 +85,7 @@ def create_pool(self, *args, **kwargs):
             n_cpu_process=self.args.cpu_procs,
             n_net_process=self.args.net_procs or self.args.io_procs,
             spill_dirs=self.args.spill_dir or os.environ.get('MARS_SPILL_DIRS'),
+            lock_free_io=self.args.lock_free_io or bool(int(os.environ.get('MARS_LOCK_FREE_IO', '0'))),
```

Comment:
Shall we collect all environment variables together to make it more maintainable?